api_app/db/migrations/airlock.py

```diff
@@ -0,0 +1,33 @@
+from azure.cosmos import CosmosClient
+from db.repositories.airlock_requests import AirlockRequestRepository
+
+
+class AirlockMigration(AirlockRequestRepository):
+    def __init__(self, client: CosmosClient):
+        super().__init__(client)
+
+    def add_created_by_and_rename_in_history(self) -> int:
+        num_updated = 0
+        for request in self.container.read_all_items():
+            # Only migrate if createdBy isn't present
+            if 'createdBy' in request:
+                break
```

Comment:
think you want `continue` here to move and try the next request?

Reply:
Spot on, thank you - updated